GameServer/Logic/GameObjects/Champion.cs

```diff
@@ -124,7 +124,7 @@ public int getTeamSize()
                 }
             }
 
-            var playerIndex = "player" + playerId;
+            var playerIndex = "player" + _playerId;
```

Comment:
This piece of code is used in two separate places, so I'd like to see it moved to a function.
```CS
string GetPlayerIndex()
{
    return string.Format("player{0}", _playerId);
}
```